### PR TITLE
suppress livemetrics service from being instrumented

### PR DESCRIPTION
**Pull request by TimothyMothra**
use `SuppressInstrumentationScope.Begin()` to prevent Ping and Post from being instrumented. Set this once, when the dedicated thread starts.

### CODE REVIEW

File: sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Customizations/QuickPulseSDKClientAPIsRestClient.cs [hunk 1]
```diff
@@ -36,6 +37,9 @@ public QuickPulseResponse CustomPing(string ikey, string apikey = null, int? xMs
                 throw new ArgumentNullException(nameof(ikey));
             }
 
+            // Prevents the http operations from being instrumented.
+            using var scope = SuppressInstrumentationScope.Begin();
```
Comment (reyang) [inline sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Customizations/QuickPulseSDKClientAPIsRestClient.cs:41]: Why would this happen here during each call instead of set once on the dedicated thread?

Comment (TimothyMothra) [reply sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Customizations/QuickPulseSDKClientAPIsRestClient.cs:41]: hmmm. I like that better. 
I initially thought to suppress the smallest scope possible, but I was concerned about the allocations.
Your suggestion should be much better, i'll make the change.

File: sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Internals/Manager.State.cs [hunk 2]
```diff
@@ -110,6 +111,9 @@ private void Run()
         {
             try
             {
+                // Prevents the http operations from being instrumented.
```
Comment (reyang) [inline sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Internals/Manager.State.cs:114]: How should I interpret this comment? (it doesn't seem to be related to HTTP at all)

Comment (TimothyMothra) [reply sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Internals/Manager.State.cs:114]: rewrote to be more explicit: `// Suppress the outbound Live Metrics service calls from being collected as dependency telemetry.`

File: sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Internals/Manager.State.cs [hunk 3]
```diff
@@ -110,6 +111,9 @@ private void Run()
         {
             try
             {
+                // Suppress the outbound Live Metrics service calls from being collected as dependency telemetry.
+                using var scope = SuppressInstrumentationScope.Begin();
```
Comment (rajkumar-rangaraj) [inline sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Internals/Manager.State.cs:115]: Should we move it here? - https://github.com/Azure/azure-sdk-for-net/blob/b4cec1cd78f765907fffd327f0ca2c39710b1871/sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Internals/Manager.cs#L29

**Review by rajkumar-rangaraj — APPROVED** [review]
